docs/docs-site/content/developer/api/rest/_index.md

```diff
@@ -492,6 +492,86 @@ Create a directory at a specific path:
 - **name:** name of the directory
 - **path:** specific path where user wants to create the directory
 
+### Create File
+
+Create a file at a specific path:
+
+    curl -X POST https://demo.gethue.com/api/v1/storage/touch
+    
+- **name:** name of the file
+- **path:** specific path where user wants to create the file
```

Comment:
Global comment:
I'd prefer to use the second person, i.e. "you" instead of "user".
For example: A specific path where you want to create the file